service/trino-verifier/src/main/java/io/trino/verifier/VerifierQueryEventDao.java

```diff
@@ -39,7 +39,7 @@
             "  control_teardown_query_ids_json VARCHAR(255) NULL,\n" +
             "  control_cpu_time_seconds DOUBLE NULL,\n" +
             "  control_wall_time_seconds DOUBLE NULL,\n" +
-            "  error_message TEXT NULL,\n" +
+            "  error_message MEDIUMTEXT NULL,\n" +
```

Comment:
why not `longtext`?

Reply:
No real reason. But 16MB should be enough for everybody ;)